components/admin_console/billing/payment_info_details.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import CardImage from 'components/payment_form/card_image';
+import {GlobalState} from 'types/store';
+
+export interface PaymentInfoDetailsProps {
+    children?: React.ReactNode;
+}
+
+const PaymentInfoDetails: React.FC<PaymentInfoDetailsProps> = ({children}: PaymentInfoDetailsProps) => {
```

Comment:
Nit: payment_info_details (PaymentInfoDetails), "info" and "details" mean basically the same thing. Can we shorten this to payment_info or payment_details? I think it would be a little nicer for readability

Reply:
ok.